src/base1/cockpit.js

```diff
@@ -623,7 +628,9 @@ function Transport() {
             /* 'ping' messages are ignored */
 
         } else if (data.command == "hint") {
-            if (process_hints)
+            if (data.hint == "ignore-health-check")
```

Comment:
Control messages from a frame towards the parent are not processed here, only control messages from the parent to the frame.